src/libLookup/Lookup.cpp

```diff
@@ -1487,10 +1487,6 @@ bool Lookup::ProcessSetDSBlockFromSeed(const vector<unsigned char>& message,
 
   LOG_MARKER();
 
-  if (AlreadyJoinedNetwork()) {
```

Comment:
What's the intention of putting this down?

Reply:
because it will never check the condition if SyncTyepe == NoSync and return from here.

Reply:
The original intention for this check is for discarding the extra ProcessSetDSBlock message

Reply:
yes, it would still discard it but after notifying the cv

Reply:
ok I see